### PR TITLE
Adding PROFILE.md

### PR DESCRIPTION
**Pull request by nathanielmrsh**
### Why:

Closes [issue link]

### Check off the following:

- [ ] For workflow changes, I have verified the Actions workflows function as expected.
- [ ] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by rozbecoz — APPROVED** [review]
reviewing myself : P

**Comment by rozbecoz** [comment]
what the hell

**Comment by rozbecoz** [comment]
oh, wait, I'm on the wrong profile ...sorry

File: PROFILE.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Welcome to my Github profile
```
Comment (Mr-active) [inline PROFILE.md:1]: Duplicate of #

**Review by Mr-active — COMMENTED** [review]
<upload>
https://github.com/skills/introduction-to-github/pull/257/files#r1087333301

**Pull request closed without merging**